GVFS/GVFS.Hooks/GVFS.Hooks.Mac.csproj

```diff
@@ -93,18 +93,23 @@
     <Compile Include="..\GVFS.Common\Tracing\Keywords.cs">
       <Link>Common\Tracing\Keywords.cs</Link>
     </Compile>
-    <Compile Include="..\GVFS.Platform.Mac\MacPlatform.Shared.cs" Link="MacPlatform.Shared.cs" />
     <Compile Include="..\GVFS.Platform.POSIX\POSIXFileSystem.Shared.cs">
       <Link>POSIX\POSIXFileSystem.Shared.cs</Link>
     </Compile>
     <Compile Include="..\GVFS.Platform.POSIX\POSIXPlatform.Shared.cs">
       <Link>POSIX\POSIXPlatform.Shared.cs</Link>
     </Compile>
+    <Compile Include="..\GVFS.Platform.Mac\MacPlatform.Shared.cs">
+      <Link>Mac\MacPlatform.Shared.cs</Link>
+    </Compile>
   </ItemGroup>
   
   <ItemGroup>
     <PackageReference Include="StyleCop.Analyzers" Version="1.0.2">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Mac\" />
+  </ItemGroup>

```

Comment:
Why was this added?

```
  <ItemGroup>
    <Folder Include="Mac\" />
  </ItemGroup>
```

Can it be removed?

Reply:
I think VS added it when I linked in `MacPlatform.Shared`. I'll remove it and see what happens.